monasca-alarms/build.yml

```diff
@@ -1,7 +1,7 @@
 repository: monasca/alarms
 variants:
-  - tag: 1.1.3
+  - tag: 1.1.4
     aliases:
       - :latest
     args:
-      MON_CLIENT_VERSION: "1.6.0"
+      MON_CLIENT_VERSION: "1.7.0"
```

Comment:
Let's change the client version to **1.8.0** here :)

Reply:
It seems this isn't going to work quite yet: https://hub.docker.com/r/monasca/client/tags/

manifest for monasca/client:1.8.0 not found

Reply:
Looks like tagging 1.8 failed:
https://travis-ci.org/monasca/monasca-docker/jobs/295171347

Reply:
And after merging #313 https://travis-ci.org/monasca/monasca-docker/jobs/295486366
https://storage.googleapis.com/monasca-ci-logs/monasca-docker%2F2017-11-01_15-26-15_monasca_logs%2Fbuild%2F2017-11-01-15-26-16-monasca-client-1.8.0.log.gz
```
+ getopt -n parse-options -o r:b:e:d:c:u:q: -- -r https://git.openstack.org/openstack/python-monascaclient -b 1.8.0 -q master
+ OPTS= -r 'https://git.openstack.org/openstack/python-monascaclient' -b '1.8.0' -q 'master' --
+ echo Failed parsing options.
Failed parsing options.
+ exit 1
```

Reply:
It is probably because we need https://travis-ci.org/monasca/monasca-docker/builds/295486362 to pass, should go through soon though